app/src/main/java/org/mozilla/focus/engine/EngineViewLifecycleFragment.kt

```diff
@@ -51,24 +46,30 @@ abstract class EngineViewLifecycleFragment : LocaleAwareFragment() {
 
     open fun onWebViewCreated(webView: EngineView) = Unit
 
-    override fun onPause() {
-        webView!!.pauseTimers()
-        super.onPause()
-    }
-
-    override fun onResume() {
-        webView!!.resumeTimers()
-        super.onResume()
-    }
-
     override fun onStop() {
         super.onStop()
-        webView!!.onStop() // internally calls WebView.onPause: see impl for details.
+
+        // NB: onStop unexpectedly calls onPause: see below.
+        //
+        // When the user says "Alexa pause [the video]", the Activity will be paused/resumed while
+        // Alexa handles the request. If the WebView is paused during video playback, the video will
+        // have poor behavior (on YouTube the screen goes black, may rebuffer, and may lose the voice
+        // command). Unfortunately, there does not appear to be any way to prevent this other than
+        // to not call WebView.onPause so we pause the WebView later, here in onStop, when it isn't
+        // affected by Alexa voice commands. Luckily, Alexa pauses the video for us. afaict, on
+        // Fire TV, `onPause` without `onStop` isn't called very often so I don't think there will
+        // be many side effects (#936).
+        //
+        // The problem is not reproducible when onPause is called here, even if pauseTimers is
+        // called in onPause.
```

Comment:
Nit: If the `webview.pauseTimers()` is no longer necessary (I assume?), the reference to it in this comment is a bit confusing.  Can we remove it?